qa/tasks/cephfs/test_admin.py

```diff
@@ -2921,3 +2921,59 @@ def test_enable_post_disabling(self):
         # plugin is reported properly by "ceph mgr module ls" command, check if
         # it is also working fine.
         self.run_ceph_cmd('fs volume ls')
+
+
+class TestTemp(TestAdminCommands):
```

Comment:
Will rename this or move the tests underneath it to a different class eventually.